### DISCUSSION TITLE
Error Codes

### DISCUSSION THREAD

**Discussion by emassen** [mikopbx/Core#382] [General ( Обо всём )]
Hi,

A fix was made for the busy extension but it still didn't work. On Yealink phone instead of showing the error name only the error code appears. As follows below.

BUSY = DISPLAY YEALINK PHONE = Q.850;cause=17
UNAVAILABLE = DISPLAY YEALINK PHONE = Q.850;cause=16
THAT DOES NOT EXIST =  DISPLAY YEALINK PHONE = Q.850;cause=1

Thanks

**Comment by boffart** [comment]
See disussion:
https://support.yeastar.com/hc/en-us/community/posts/4414220030873-BYE-Q-850-cause-16
And docs:
https://www.yealink.com/upfiles/products/201812/1543804211178.pdf

You should try to edit the parameter:
**sip.call_fail_use_reason.enable**

It enables or disables the
phone screen to display
the reason carried in the
Reason header of SIP
response when the phone
fails to dial out

**Comment by emassen** [comment]
Hi,

Thank you very much, problem fixed. It was really on the phones. I thought it was on the PBX. Sorry for the inconvenience.
